composer.json

```diff
@@ -32,7 +32,7 @@
         "psr/log": "^1.0|^2.0|^3.0",
         "psr/simple-cache": "^1.0|^2.0|^3.0",
         "ramsey/uuid": "^4.2.2",
-        "symfony/console": "^6.0",
+        "symfony/console": "^6.0.3",
```

Comment:
Bumped this as there is a bump in lower versions parsing escaped spaces in arguments, which was breaking my test for an escaped value.